Wikipedia/Code/NotificationsCenterViewModel.swift

```diff
@@ -157,7 +155,6 @@ final class NotificationsCenterViewModel: NSObject {
             switch result {
             case .failure(let error):
                 DDLogError("Error refreshing notifications: \(error)")
-                //TODO: show some sort of error state
```

Comment:
These will be addressed in https://phabricator.wikimedia.org/T288773 and https://phabricator.wikimedia.org/T287766.